libdeno/internal.h

```diff
@@ -99,7 +101,7 @@ class DenoIsolate {
   v8::Persistent<v8::Function> recv_;
   v8::StartupData snapshot_;
   v8::Persistent<v8::ArrayBuffer> global_import_buf_;
-  v8::Persistent<v8::SharedArrayBuffer> shared_ab_;
+  v8::Global<v8::SharedArrayBuffer> shared_ab_;
```

Comment:
Are you sure? I think I had problems with this before.

Reply:
Oh this is a rebase error I think. Having no problems with it atm though.